### PR TITLE
Split versions

### PR DESCRIPTION
**Pull request by trask**
see #553 and #563

### CODE REVIEW

File: dependencyManagement/build.gradle.kts [hunk 1]
```diff
@@ -12,17 +12,19 @@ val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
 // this line is managed by .github/scripts/update-sdk-version.sh
-val otelVersion = "1.19.0"
+val otelSdkVersion = "1.19.0"
+// this line is managed by .github/scripts/update-instrumentation-version.sh
+val otelInstrumentationVersion = "1.19.0"
```
Comment (trask) [inline dependencyManagement/build.gradle.kts:17]: intentionally left this at 1.19.0 to see if dependabot picks it up

... and if it doesn't, then I'll use it to test out updated workflow

**Review by jack-berg — APPROVED** [review]

File: .github/scripts/update-instrumentation-version.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+version=$1
+
+sed -Ei "s/val otelInstrumentationVersion = \"[^\"]*\"/val otelInstrumentationVersion = \"$version\"/" dependencyManagement/build.gradle.kts
```
Comment (jack-berg) [inline .github/scripts/update-instrumentation-version.sh:5]: Didn't check, but is this file executable?

Comment (jack-berg) [reply .github/scripts/update-instrumentation-version.sh:5]: Checked and it is. 👍

Comment (trask) [reply .github/scripts/update-instrumentation-version.sh:5]: thanks for checking, I usually forget until it fails during the release process, but somehow remembered this time 😂

**Pull request merged**